src/HtmlGenerator/Smart.Design.HtmlGenerator/Models/CommandResponse.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Smart.Design.HtmlGenerator.Models
+{
+    public class CommandResponse
+    {
+        private string SuccessMessage;
+
+        private List<Exception> Errors = new();
+
+        public bool IsSuccess => !Errors.Any();
+
+        public string GetResponseMessage()
+        {
+            if (IsSuccess) return SuccessMessage;
+
+            StringBuilder stringBuilder = new("Exception Stacks");
+            Errors.ForEach(error => stringBuilder.AppendLine((string?) $"exception with message => {error}"));
```

Comment:
```suggestion
            Errors.ForEach(error => stringBuilder.AppendLine((string?) $"Exception with message => {error}"));
```

Reply:
Shouldn't be a breakline also between "Exception stack" and the error lines ?